packages/core/src/compose-tunnel-agent-client.ts

```diff
@@ -70,6 +72,10 @@ export const addComposeTunnelAgentService = (
     machineStatusCommand?: MachineStatusCommand
     envMetadata: EnvMetadata
     composeModelPath: string
+    composeProject: string
+    profileThumbprint?: string
+    privateMode: boolean
```

Comment:
The privateMode can be derived from the defaultAccess paramter (or the opposite), it's a label with product semantics, while defaultAccess is a CTA env var, but they are basically the same. (The label is currently only read by the Docker extension AFAIK)

Reply:
can the `privateMode` be read from the `/env-metadata` endpoint instead of the label?